deploy/helm/edot-collector/kube-stack/README.md

```diff
@@ -0,0 +1,63 @@
+## Kube-stack Helm Chart
+
+**A more detailed documentation can be found [here](https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md).**
+
+The [kube-stack Helm Chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack) is used to manage the installation of the OpenTelemetry operator (including its CRDs) and to configure a suite of EDOT collectors, which instrument various Kubernetes components to enable comprehensive observability and monitoring.
+
+The chart is installed with a provided default `values.yaml` file that can be customized when needed.
+
+### DaemonSet collectors
+
+The OpenTelemetry components deployed within the DaemonSet EDOT collectors are responsible for observing specific signals from each node. To ensure complete data collection, these components must be deployed on every node in the cluster. Failing to do so will result in partial and potentially incomplete data.
+
+The DaemonSet collectors handle the following data:
+
+- Host Metrics: Collects host metrics (hostmetrics receiver) specific to each node.
+- Kubernetes Metrics: Captures metrics related to the Kubernetes infrastructure on each node.
+- Logs: Utilizes a filelog receiver to gather logs from all Pods running on the respective node.
+- OTLP Traces Receiver: Opens an HTTP and a GRPC port on the node to receive OTLP trace data.
+
+### Deployment collector
+
+The OpenTelemetry components deployed within a Deployment collector focus on gathering data at the cluster level rather than at individual nodes. Unlike DaemonSet collectors, which need to be deployed on every node, a Deployment collector operates as a standalone instance.
```

Comment:
```suggestion
The OpenTelemetry components deployed within a Deployment collector focus on gathering data at the cluster level rather than at individual nodes.  A Deployment instance of the collector operates as a standalone (unlike DaemonSet collector instances, which are deployed on every node)
```